### ISSUE TITLE
SetVehicleSpawnInfo delay & interiorid

### ISSUE THREAD

**Issue by BeckzyBoi** [openmultiplayer/open.mp#655]
When used, purely as example:

`public OnGameModeInit()
{
    new vehid = AddStaticVehicleEx(400, 0.0, 0.0, 10.0, 0.0, 0, 0, 300);
    SetVehicleSpawnInfo(vehid, 401, 0.0, 0.0, 10.0, 0.0, 0, 0, 300, -2);
    SetVehicleToRespawn(vehid);
}`

The vehicle model is changed to 401, and the respawn delay is/should still be 300, while the interior ID should also remain the same; however the vehicle becomes invisible (not interior ID 0?).

When keeping the respawn delay the same using -2, the vehicle appears as it should:

`SetVehicleSpawnInfo(vehid, 401, 0.0, 0.0, 10.0, 0.0, 0, 0, -2, -2);`

Basically, it appears that using anything other than -2 for the delay, OR using the same delay by definition, then using -2 to keep the same interiorid, won't correctly set the interiorid.

I can do some further digging if required. Thanks for your time.

**Comment by AmyrAhmady** [comment]
Confirmed to be fixed in next build by OP
